nemo/collections/asr/models/ctc_models.py

```diff
@@ -365,7 +365,7 @@ def _setup_dataloader_from_config(self, config: Optional[Dict]):
             return dataset
 
         shuffle = config['shuffle']
-        if config.get('is_tarred', False):
+        if isinstance(dataset, torch.utils.data.IterableDataset):
```

Comment:
Same as above

Reply:
Dataset cannot be None. There is the following code just above the line you have highlighted:

```python
if dataset is None:
    return None
```